packages/base/src/hooks/useSyncRef.ts

```diff
@@ -0,0 +1,19 @@
+import { MutableRefObject, Ref, RefCallback, RefObject, useRef } from 'react';
+
+export function useSyncRef<RefType = any>(ref: Ref<RefType>): [RefCallback<RefType>, RefObject<RefType>] {
+  const localRef = useRef<RefType>();
```

Comment:
```suggestion
  const localRef = useRef<RefType>(null);
```
I guess this should be called like this to return a `MutableRefObject`